homeworks/task_2.py

```diff
@@ -0,0 +1,27 @@
+# 2.Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+# Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого
+# отрезка-стороны с суммой двух других. Если хотя бы в одном случае отрезок окажется больше
+# суммы двух других, то треугольника с такими сторонами не существует. Отдельно сообщить
+# является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+NUMBER_OF_SIDES_TRIANGLE = 3
+while True:
+    str_input = input('Введите длины сторон треугольника a b c (через пробел): ').split(' ')
```

Comment:
Если хотите разбить строку по пробелам, то лучше ничего не передавать методу `split`
```suggestion
    str_input = input('Введите длины сторон треугольника a b c (через пробел): ').split()
```

Reply:
Спасибо, внес исправления